app/code/pagetype/HomePage.php

```diff
@@ -61,20 +82,89 @@ public function getCMSFields()
         $fields = parent::getCMSFields();
 
         // Banner Tab
-        $fields->addFieldToTab('Root.Banner', TextField::create(
-            'BannerHeading',
-            'Banner Heading'
-        ));
-
-        $fields->addFieldToTab('Root.Banner', TextField::create(
-            'BannerDesc',
-            'Banner Description'
-        ));
-
-        $fields->addFieldToTab('Root.Banner', $bannerImage = UploadField::create(
-            'BannerImage',
-            'Banner Image'
-        )->setDescription('Only supports <strong>jpg, jpeg, png</strong> filetypes.</br>Recommended dimensions 1920 x 1080 px.'));
+        $fields->addFieldToTab(
+            'Root.Banner',
+            TextField::create(
+                'BannerHeading',
+                'Banner Heading'
+            )
+        );
+
+        $fields->addFieldToTab(
+            'Root.Banner',
+            TextField::create(
+                'BannerDesc',
+                'Banner Description'
+            )
+        );
+
+        $fields->addFieldToTab(
+            'Root.Banner',
+            $bannerImage = UploadField::create(
+                'BannerImage',
+                'Banner Image'
+            )->setDescription('Only supports <strong>jpg, jpeg, png</strong> filetypes.</br>Recommended dimensions 1920 x 1080 px.')
+        );
+
+        // Accomodaiton configurations
```

Comment:
slight typo on accomodation ;)

Reply:
As mentioned above, you can apply the same refactoring concept of `addFieldsToTab` for the accomodation tab :)